api/src/main/java/org/pmiops/workbench/compliancetraining/ComplianceTrainingServiceImpl.java

```diff
@@ -219,6 +223,8 @@ public DbUser syncComplianceTrainingStatusMoodle(DbUser dbUser, Agent agent)
 
   @Transactional
   public DbUser syncComplianceTrainingStatusAbsorb(DbUser dbUser, Agent agent) throws ApiException {
+    log.info("Using Absorb to sync compliance training status.");
```

Comment:
This is a sanity check on our `useAbsorb` logic to see if the user is actually querying Absorb. It can be removed when we eventually remove Moodle. IMO these are the least important logs added in this PR.